waltid-libraries/waltid-did/src/commonMain/kotlin/id/walt/did/dids/resolver/local/DidWebResolver.kt

```diff
@@ -127,7 +126,16 @@ class DidWebResolver(private val client: HttpClient) : LocalResolverMethod("web"
     }
 
     companion object {
-        const val URL_PROTOCOL = "https"
+
+        var urlProtocol = "https"
```

Comment:
since `urlProtocol` is a static public variable, I could update it from anywhere (just by class name), overwriting the effects of `enableHttps`
it probably needs to be private, or perhaps reconsider the logic, e.g. use a computed property:
```kotlin
var httpsEnabled: Boolean = true

val urlProtocol: String
    get() = if (httpsEnabled) "https" else "http"
```